spec/controllers/application_controller_spec.rb

```diff
@@ -0,0 +1,163 @@
+require 'spec_helper'
+
+describe ApplicationController do
+
+
+  it 'loads the homepage' do 
+    get '/'
+    expect(last_response.status).to eq(200)
+    expect(last_response.body).to include("Welcome to Fwitter")
+  end
+
+  it 'loads the signup page' do 
+    get '/signup'
+    expect(last_response.status).to eq(200)
+  end
+
+  it 'signup directs user to twitter index' do 
+    params = {
+      :username => "skittles123",
+      :email => "skittles@aol.com",
+      :password => "rainbows"
+    }
+    post '/signup', params
+    expect(last_response.location).to eq("http://example.org/tweets") 
```

Comment:
this may confuse people a bit that it's example.org. CAn you just compare to `/tweets`?